src/design-system/components/Text/useTextStyle.ts

```diff
@@ -26,6 +26,16 @@ export function useTextStyle({
           .join(', ')}`
       );
     }
+
+    if (!textSizes[size]) {
```

Comment:
Love this. Do we need an `if (__DEV__) {` check like you did above?

Reply:
This is actually nested inside one large `if (__DEV__) {` block, it's just not visible in this diff—but I can see that makes it less clear so I might update it.

Reply:
🤦 my b!